content/_help/verify-your-identity/how-to-take-photos-to-verify-your-identity._zh.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: help
+title: 如何拍照来验证身份
+category: verify-your-identity
+permalink: /zh/help/verify-your-identity/how-to-take-photos-to-verify-your-identity/
+order: 3
+redirect_from: 
+  - /zh/help/verify-your-identity/troubleshoot-uploading-your-state-issued-id/
+  - /zh/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/
+url_safari: https://www.apple.com/cn/safari/
+url_chrome: https://www.google.com/chrome/?brand=WDIF&ds_kid=43700078347700321&gad_source=1&gclid=CjwKCAjww_iwBhApEiwAuG6ccAvZWVPqrBawjLCJp6uWvrMplezDwWVR7AnWXZhu-4He4V3oXJBOrRoCtTwQAvD_BwE&gclsrc=aw.ds&hl=zh-cn
+url_safari_update_doc: https://support.apple.com/zh-cn/102665
+url_safari_clear_cache_doc: https://support.apple.com/zh-cn/105082
+url_chrome_update_doc: https://support.google.com/chrome/answer/95414?co=GENIE.Platform%3DDesktop&hl=zh-Hans
+url_chrome_clear_cache_doc: https://support.google.com/accounts/answer/32050?co=GENIE.Platform%3DAndroid&oco=1&hl=zh-Hans
+---
+
+{% capture alert_link %}<a href="/zh/help/verify-your-identity/accepted-identification-documents/">有效驾照或州颁发的身份证件</a>{% endcapture %}
+
+{% capture alert_content %}
+  <strong>
+    如果你没有{{ alert_link }}就不能用 Login.gov 进行身份验证。
+  </strong>
+  请联系我们合作机构的帮助中心来了解你该怎么办。
+{% endcapture %}
+
+{%
+  include alert.html
+  content=alert_content
+  type='error'
+  role='alert'
+%}
+
+## 概述
+
+要在网上验证身份，您必须同时完成以下操作：
+ 
+* 拍一张您有效的驾驶执照或州身份卡的照片，并且
+* 输入您的个人信息
+
+您可能还需要使用手机或平板电脑拍张自己的照片，以显示您就是身份证件上的人。如果您没有带相机的手机或平板电脑，您可以亲身去验证身份。
```

Comment:
The figma shows "verify your identity in person" as a link

This is my best guess but it is just that, a guess. The closest thing I see partially matching the link in the troubleshooting section, 选择**亲身验证身份**的选项,  is **亲身**去**验证身份**. I don't know the significance of the inserted character (去) or what the full context of the troubleshooting text is.

I'm suggesting grabbing everything after the comma since we know that is a complete phrase.

```suggestion
您可能还需要使用手机或平板电脑拍张自己的照片，以显示您就是身份证件上的人。如果您没有带相机的手机或平板电脑，[您可以亲身去验证身份](/zh/help/verify-your-identity/verify-your-identity-in-person/)。
```

Reply:
This is nearly perfect -- it might be good to just omit "您可以" in the link, leaving just 您可以[亲身去验证身份]

Reply:
Dang it - I had that then changed it! :-P

```suggestion
您可能还需要使用手机或平板电脑拍张自己的照片，以显示您就是身份证件上的人。如果您没有带相机的手机或平板电脑，您可以[亲身去验证身份](/zh/help/verify-your-identity/verify-your-identity-in-person/)。
```